src/snyk/common/views/issueTreeProvider.ts

```diff
@@ -136,6 +158,10 @@ export abstract class ProductIssueTreeProvider<T> extends AnalysisTreeNodeProvid
           totalVulnCount++;
           folderVulnCount++;
 
+          if (this.getAIFix(issue)) {
+            AIFixVulnCoun++;
+          }
```

Comment:
this should be somewhere else or rename the function to represent it accordingly

the return type also should not be a tuple